client/dfclient/src/gameplay_state.cpp

```diff
@@ -463,6 +463,9 @@ void GameplayState::OnKeyPressed(const ncine::KeyboardEvent &event) {
 	
 	if (event.sym == ncine::KeySym::TAB)
 		this->showHighscores = true;
+
+	if (event.sym == ncine::KeySym::ESCAPE)
```

Comment:
Please add an additional dialog or remove this, quitting with just escape could be triggered by accident.